tests/Twilio/Integration/Wireless/V1/UsageRecordTest.php

```diff
@@ -26,7 +26,10 @@ public function testReadRequest(): void {
 
         $this->assertRequest(new Request(
             'get',
-            'https://wireless.twilio.com/v1/UsageRecords'
+            'https://wireless.twilio.com/v1/UsageRecords',
```

Comment:
Are these changes needed?

Reply:
Not necessarily. I can add additional logic to prevent these changes. I wasn't sure if it mattered since these are just integration tests.

Reply:
Meh. It's fine as-is.